### PR TITLE
export bin/ commands

### PR DESCRIPTION
**Pull request by fbarotov**

### CODE REVIEW

**Comment by fbarotov** [comment]
Part of the fix for the homebrew https://github.com/hazelcast/homebrew-hz/pull/16

**Comment by Holmistr** [comment]
We should stop asking @alparslanavci for the reviews unless we don't know what to do. The command line packages are going under Core team maintenance. 
Let's ask @frant-hartm to take a look.

File: packages/deb/hazelcast/DEBIAN/postinst [hunk 1]
```diff
@@ -1,5 +1,15 @@
 #!/bin/sh
 echo 'hazelcastDownloadId=CLI_DEB' > "/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/download/hazelcast-download.properties"
 
+for filename in /usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hz*; do
+  case "$filename" in
+    *bat)
+      ;;
+    *)
+      ln -s "$filename" /usr/bin/"$(basename "$filename")"
```
Comment (frant-hartm) [inline packages/deb/hazelcast/DEBIAN/postinst:9]: Do we need to remove this in some post uninstall script?

**Review by frant-hartm — CHANGES_REQUESTED** [review]

File: packages/deb/hazelcast/DEBIAN/postinst (hunk 1 above)
Comment (fbarotov) [reply packages/deb/hazelcast/DEBIAN/postinst:9]: @frant-hartm added symlink removal in post uninstall phases, can you please re-review?

**Review by frant-hartm — APPROVED** [review]
LGTM now.

**Pull request merged**